src/lib/utils/ghash/ghash.cpp

```diff
@@ -97,6 +97,18 @@ void GHASH::key_schedule(std::span<const uint8_t> key) {
    BOTAN_ASSERT_NOMSG(key.size() == GCM_BS);
    auto H = load_be<std::array<uint64_t, 2>>(key.first<GCM_BS>());
 
+#if defined(BOTAN_HAS_GHASH_CLMUL_CPU)
+   if(CPUID::has_carryless_multiply()) {
+      zap(m_HM);
+      if(m_H_pow.size() != 8) {
+         m_H_pow.resize(8);
+      }
```

Comment:
I'm fairly sure that [resize is a noop](https://en.cppreference.com/w/cpp/container/vector/resize) when the buffer is already of that size, no? Or am I missing something?

Reply:
I think that is true, just a nervous tic on my part